### PR TITLE
Milestone 4  - Process Data Models

### PR DESCRIPTION
**Pull request by sneeu-leeu**
## In this Milestone I:
-  Set up associations for `user`, `post`, and `like`
- Add Methods:
1) Show Recent Posts for user
2) Show recent 5 comments for post
3) Update Posts Counter
4) Update Comments counter
5) Update Likes Counter

### CODE REVIEW

File: app/models/comment.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user, class_name: 'User'
+  belongs_to :post
+
+  def update_comments_counter
+    user.increment!(:comments_counter)
+  end
```
Comment (ShoiraTa) [inline app/models/comment.rb]: - [x] Comment model should have a method that updates the comments counter for a post, in your case it is user 😉 :
```ruby
def update_comments_counter
  post.increment!(:comments_counter)
end
```

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @sneeu-leeu ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Models are properly setup ✔️ 
- The associations between models are set up correctly according to the given ✔️ 
- Professional pull request! ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/comment.rb (hunk 1 above)
Comment (sneeu-leeu) [reply app/models/comment.rb]: Thanks for catching that!

**Review by arslanbisharat — APPROVED** [review]
Hi,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**